packages/core/src/models/base.ts

```diff
@@ -47,19 +47,20 @@ export abstract class BaseModel<T extends SVGElement> {
   getMousePosition(event: PointerEvent): Point {
     const el = this.drauu.el!
     const scale = this.drauu.options.coordinateScale ?? 1
+    const position = this.drauu.options.coordinatePosition ?? { x: 0, y: 0 }
 
     if (this.drauu.options.coordinateTransform === false) {
       const rect = this.drauu.el!.getBoundingClientRect()
       return {
-        x: (event.pageX - rect.left) * scale,
-        y: (event.pageY - rect.top) * scale,
+        x: (event.pageX - rect.left - position.x) * scale,
+        y: (event.pageY - rect.top - position.y) * scale,
```

Comment:
```suggestion
        x: (event.pageX - rect.left + origin.x) * scale,
        y: (event.pageY - rect.top + origin.y) * scale,
```

Reply:
Do you think the origin should be added instead of subtracted?

Just to give you an example, today I'm using it this way:
```

<template>
  <div class="w-layer draw-zone">
    <svg
      ref="initialDrawCanvasRef"
      class="drawer-zone__canvas"
      preserveAspectRatio="none"
    ></svg>

    <slot />
  </div>
</template>

<script setup lang="ts">
/* eslint-disable import/no-unresolved */
import { ref, watchEffect, computed } from 'vue'
import { toRefs, whenever } from '@vueuse/core'
import { useWhiteBoard } from '../composables'

const whiteboard = useWhiteBoard()
const { state, setDrawCanvasRef, drawManager } = whiteboard
const initialDrawCanvasRef = ref<SVGGraphicsElement>()

const drawCanvasTransform = computed(
  () =>
    `translate(${state.panel.canvasTranslate.x}px, ${state.panel.canvasTranslate.y}px) scale(${state.panel.currentZoom})`,
)

const { drauuInstance } = toRefs(drawManager.drauu)

whenever(
  () => initialDrawCanvasRef.value,
  (value) => {
    setDrawCanvasRef(value)
  },
)

watchEffect(() => {
  if (
    state.panel.currentZoom &&
    whiteboard.drawCanvasRef &&
    drauuInstance.value?.options &&
    whiteboard.element &&
    state.panel.canvasTranslate
  ) {
    drauuInstance.value.options.coordinateScale = 1 / state.panel.currentZoom
    drauuInstance.value.options.eventTarget = whiteboard.element
    drauuInstance.value.options.coordinatePosition = {
      x: state.panel.canvasTranslate.x,
      y: state.panel.canvasTranslate.y,
    }
  }
})
</script>

<style lang="scss">
.w-board {
  & .draw-zone {
    width: 100%;
    height: 100%;
  }

  & .drawer-zone_wrapper {
    z-index: 0;
    position: absolute;
    left: 0px;
    top: 0px;
    height: 100%;
    width: 100%;
  }

  & .drawer-zone__canvas {
    position: absolute;
    left: 0px;
    top: 0px;
    height: 100%;
    width: 100%;
    & * {
      transform: v-bind('drawCanvasTransform');
    }
  }
}
</style>

```

Reply:
```vue
  const drauu = useDrauu(drawCanvasRef, {
    brush: {
      mode: 'stylus',
      color: state.drawConfig.lineColor,
      size: 3,
    },
    coordinateScale: state.panel.currentZoom,
    coordinateTransform: true,
    eventTarget: toValue(element),
  })
```

Reply:
Naming Offset actually makes sense with what happens and I understand the point of addition rather than subtraction.
Thanks for commenting!
I updated the code